samples/nextjs/src/lib/dictionary-service-factory.ts

```diff
@@ -1,20 +1,44 @@
-import { DictionaryService, GraphQLDictionaryService } from '@sitecore-jss/sitecore-jss-nextjs';
+import {
+  DictionaryService,
+  RestDictionaryService,
+  GraphQLDictionaryService,
+} from '@sitecore-jss/sitecore-jss-nextjs';
 import config from 'temp/config';
 
-export class DictionaryServiceFactory {
```

Comment:
I think we should stick to the single factory `DictionaryServiceFactory` (following the factory pattern, it's a factory that creates a `DictionaryService` instance). The check for environment variable / returning appropriate implementation can happen inside the `create()`.

Reply:
That makes a lot more sense! Will change.